docs/docs/User Guide/Configuration/Storage/S3-Compatible Storage.md

```
@@ -21,7 +21,7 @@
 
 **Command-line:**
 
-```bash
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)